graylog2-web-interface/src/views/components/visualizations/PlotLegend.test.tsx

```diff
@@ -108,10 +108,17 @@ describe('PlotLegend', () => {
     await screen.findByText('name11');
   });
 
-  it('should hide with a single values', async () => {
+  it('should hide with a single value', async () => {
     const plotConfig = AggregationWidgetConfig.builder().series([Series.forFunction('count')]).build();
     render(<SUT chartDataProp={[{ name: 'name1' }]} plotConfig={plotConfig} />);
 
     expect(screen.queryByText('name1')).not.toBeInTheDocument();
   });
+
+  it('should not hide with a single value if configured', async () => {
+    const plotConfig = AggregationWidgetConfig.builder().series([Series.forFunction('count')]).build();
+    render(<SUT chartDataProp={[{ name: 'name1' }]} plotConfig={plotConfig} neverHide />);
+
+    expect(screen.queryByText('name1')).toBeInTheDocument();
```

Comment:
In order to make this work regardless when it is rendered, I would suggest doing this:
```suggestion
    await screen.findByText('name1');
```